packages/plugin-activedirectory-api/src/connectionResolver.ts

```
@@ -0,0 +1,21 @@
+import * as mongoose from 'mongoose';
+import { IContext as IMainContext } from '@erxes/api-utils/src';
+import { createGenerateModels } from '@erxes/api-utils/src/core';
+
+export interface IModels {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)